client/v2/autocli/flag/builder.go

```diff
@@ -28,6 +28,7 @@ const (
 	AddressStringScalarType          = "cosmos.AddressString"
 	ValidatorAddressStringScalarType = "cosmos.ValidatorAddressString"
 	ConsensusAddressStringScalarType = "cosmos.ConsensusAddressString"
+	PubkeyScalarType                 = "cosmos.Pubkey"
```

Comment:
Let's add that here: https://github.com/cosmos/cosmos-sdk//blob/433203/docs/build/building-modules/05-protobuf-annotations.md#scalar